### ISSUE TITLE
Ability to put weaver.ignore outside of the exe folder

### ISSUE THREAD

**Issue by warpech** [Starcounter/Starcounter#3402]
This is a feature request. I have a problem with easy setup for weaver.ignore when the `bin` directory is excluded from Git.

As (the .io page about Weaver](http://starcounter.io/guides/tools/weaver/#how-to-exclude-a-file-from-being-processed) explains:

> To instruct the weaver to exclude files,

> 1. Create a plain text file in the same directory as your executable.
> 2. Name it weaver.ignore (without any other extension)
> 3. Specify the name of each file on a single line in the file.

The problem is, that in all our apps, the executable directory is excluded from VCS using the following `.gitignore` rule.

```
# Build Folders (you can keep bin if you'd like, to store dlls and pdbs)
[Bb]in/
```

This makes it hard to add `weaver.ignore` to VCS. A workaround for this problem is to change the above lines to:

```
# Build Folders (you can keep bin if you'd like, to store dlls and pdbs)
**/[Bb]in/[Dd]ebug/*
**/[Bb]in/[Rr]elease/*
!weaver.ignore
```

But this requires quite a deep understanding of the project structure. It would be easier if Starcounter searched for `weaver.ignore` up in the directory tree. (similarly how does the actual `.gitignore` work).

**Comment by warpech** [comment]
I just noticed that on the same page there is an instruction how to make Visual Studio copy the `weaver.ignore` file to the output directory on every build. 

This makes my feature request less needed. I don't mind closing it, unless someone agrees this is a good idea.

**Comment by per-samuelsson** [comment]
@warpech 

Good catch, never really considered this. I'm glad that VS pattern helped you, but lets not close it until I have given it some thought. I'll assign it to me and will eventually consider it.

**Comment by warpech** [comment]
I know there is also an argument against. Reading `weaver.ignore` from upper directories might affect the app startup in an unexpected way. Remote debugging, etc, will need to consider the whole environment, not just the current directory.
